src/logic.js

```diff
@@ -0,0 +1,64 @@
+import Task from './task';
+
+const Logic = () => {
+  const createNewTask = () => {
+    const tName = document.querySelectorAll('#task_name')[0].value;
+    const tDescription = document.querySelectorAll('#task_description')[0].value;
+    const tDate = document.querySelectorAll('#task_date')[0].value;
+    const tID = document.getElementById('add-task-button').className.split(' ')[2];
+    const completedTask = false;
+    let checkedButton = ' ';
+
+    const formButtons = document.getElementsByName('priorityRadios');
+    formButtons.forEach((button) => {
+      if (button.checked) {
+        checkedButton = button.value;
+      }
+    });
+
+    const project = Task(tName, tDate, tDescription, checkedButton, tID, completedTask);
```

Comment:
Before creating new task, please apply some controls to ensure it's right fully filled, maybe minimum number of characters, or simply check it's not EMPTY